scripts/stubsabot.py

```diff
@@ -327,15 +327,16 @@ def get_github_api_headers() -> Mapping[str, str]:
 
 
 @dataclass
-class GitHubInfo:
+class GitHostInfo:
+    host: Literal["github", "gitlab"]
```

Comment:
Let's also move this into a separate TypeAlias to have one source of truth, something like `SupportedGitHosts`